packages/peregrine/package.json

```diff
@@ -5,21 +5,11 @@
     "access": "public"
   },
   "description": "The core runtime of Magento PWA",
-  "main": "dist/index.js",
+  "main": "lib/index.js",
   "scripts": {
-    "build": "concurrently --raw yarn:build:cjs yarn:build:esm",
-    "build:ci": "yarn run -s build:esm",
-    "build:cjs": "BABEL_ENV=production babel src --out-dir dist --root-mode 'upward' --source-maps",
-    "build:dev": "yarn run -s build",
-    "build:esm": "BABEL_ENV=development babel src --out-dir esm --root-mode 'upward' --source-maps",
-    "clean": "rimraf dist esm",
-    "prebuild": "yarn run -s clean",
-    "prepublishOnly": "yarn run build",
+    "clean": " ",
```

Comment:
Can remove?

Reply:
Unfortunately a limitation of `yarn workspaces run` is that there is no flag to make it ignore workspaces lacking the command. Instead, it errors on all packages where the script is not present. So for any command we want to run on every workspace, we need to have dummy scripts implemented.